doc/rules/whitespace/align_enum_values.rst

```diff
@@ -0,0 +1,41 @@
+==========================
+Rule ``align_enum_values``
+==========================
+
+Align enum values on the ``=`` operator.
```

Comment:
What if I want the opposite ?

Also, since you're doing more than just aligning (cf example 2), an option would be better
- single_space
- align
- align_single_space

Reply:
I did not understood what you mean by "the opposite"...

I agree on options. Now, are those options be really needed? I'm in favor of not complicating things if not needed. I'd rather go simple for a start an add options if issues pop's out.

What do you think?

Reply:
I don't think Php-cs-fixer are providing conflicting rules.

If someone want to change
```
case BENJI     = 1;
case ELIZABETH = 2;
case CORA      = 4;
```
to
```
case BENJI = 1;
case ELIZABETH = 2;
case CORA = 4;
```
the fixer should be able to do it

That what I meant by the opposite.
You should be able to align or de-align.

Reply:
There already exists rules for droping the spaces. If one doesn't want things aligned, one just have do not use the rule at all.

Reply:
@Vinorcola please take a look at [`phpdoc_align`](https://cs.symfony.com/doc/rules/phpdoc/phpdoc_align.html) fixer - it has both strategies available, so you can standardise the code either way (`left` / `vertical`). Your fixer should allow the same thing, because standard in the project can be changed at any point and everybody should be able to re-apply rules to the codebase.